source/builtin_macros/src/structural.rs

```diff
@@ -0,0 +1,25 @@
+use quote::quote;
```

Comment:
This is just moving the `derive_structural` proc macro to a separate module. No code changes in this function.